fuse_models/src/acceleration_2d.cpp

```diff
@@ -71,6 +70,11 @@ void Acceleration2D::onInit()
     ROS_WARN_STREAM("No dimensions were specified. Data from topic " << ros::names::resolve(params_.topic) <<
                     " will be ignored.");
   }
+
```

Comment:
You need to `#include <memory>` above in order to use `std::make_unique`. The linter is failing on that: https://build.ros.org/job/Npr__fuse__ubuntu_focal_amd64/207/testReport/junit/roslint/LintCheck/roslint_fuse_models/
```bash
/tmp/ws/src/fuse/fuse_models/src/acceleration_2d.cpp:76:  Add #include <memory> for make_unique<>  [build/include_what_you_use] [4]
```